.travis.yml

```diff
@@ -34,7 +29,8 @@ before_script:
 
 script:
 - python app/main.py >> log.txt 2>&1  &
-# - nosetests app/tests/test.py -v --with-coverage | uncommented as tests are not ready yet.
+- sleep 2
```

Comment:
Do we require this?

Reply:
Yes .. it takes some time for server to start. Hence `sleep 2`